### PR TITLE
Feature/read fixedwidth metadata files

### PR DESCRIPTION
**Pull request by kaspersorensen**
Assuming that #96 (METAMODEL-244) is going forward, here's a patch that would add another logical piece on top: A utility reader class that allows reading Fixed Width file configuration from external metadata files. I encountered two of these formats myself so felt the need for a utility and that might just as well reside in MetaModel if anyone can reuse it.

### CODE REVIEW

File: core/src/main/java/org/apache/metamodel/schema/naming/DefaultColumnNamingStrategy.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ */
+package org.apache.metamodel.schema.naming;
+
+import org.apache.metamodel.schema.builder.DelegatingIntrinsicSwitchColumnNamingStrategy;
+
+/**
+ * The default (in most cases) {@link ColumnNamingStrategy} to use when no other
+ * strategy is specified.
+ */
+public class DefaultColumnNamingStrategy extends DelegatingIntrinsicSwitchColumnNamingStrategy {
```
Comment (LosD) [inline core/src/main/java/org/apache/metamodel/schema/naming/DefaultColumnNamingStrategy.java]: That's a pretty weird way to make something a default to me.

This screams for some sort of proper factory.

Comment (LosD) [inline core/src/main/java/org/apache/metamodel/schema/naming/DefaultColumnNamingStrategy.java]: Oh damn. Probably wrong PR. I'm really starting to hate built-on-top PRs.

**Comment by LosD** [comment]
Let's wait with reviewing this until #96 has been merged and master merged here.

**Comment by kaspersorensen** [comment]
Should be ready for re-review now :)

File: core/src/main/java/org/apache/metamodel/schema/naming/CustomColumnNamingStrategy.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    private static final long serialVersionUID = 1L;
+
+    private final List<String> columnNames;
+
+    public CustomColumnNamingStrategy(List<String> columnNames) {
+        this.columnNames = columnNames;
+    }
+
+    public CustomColumnNamingStrategy(String... columnNames) {
+        this.columnNames = Arrays.asList(columnNames);
```
Comment (LosD) [inline core/src/main/java/org/apache/metamodel/schema/naming/CustomColumnNamingStrategy.java]: No biggie, but I guess best practice would be to do `this(Arrays.asList(columnNames));`

**Comment by LosD** [comment]
Single very, very minor thing. Merge if you don't care :)

**Comment by kaspersorensen** [comment]
Created METAMODEL-247 to represent this fix.

**Pull request closed without merging**